internal/controller/ols_console_ui_assets.go

```diff
@@ -234,6 +234,11 @@ func (r *OLSConfigReconciler) generateConsoleUIPlugin(cr *olsv1alpha1.OLSConfig)
 		},
 	}
 
+	// Conditionally add the CA certificate if provided in the CRD
+	if cr.Spec.OLSConfig.DeploymentConfig.ConsoleContainer.CAcertificate != "" {
+		plugin.Spec.Proxy[0].CACertificate = cr.Spec.OLSConfig.DeploymentConfig.ConsoleContainer.CAcertificate
+	}
+
```

Comment:
Do we need to update the certificate in the deployment serving UI assets, too? 
https://github.com/openshift/lightspeed-operator/blob/main/internal/controller/ols_console_ui_assets.go#L138-L143
https://github.com/openshift/lightspeed-operator/blob/main/internal/controller/ols_console_ui_assets.go#L50-L51